utils/clixon/Makefile

```diff
@@ -0,0 +1,95 @@
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILE:=LICENSE.md
+
+PKG_SOURCE_URL:=https://github.com/clicon/$(PKG_NAME)/archive/refs/tags
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL_FILE:=$(PKG_VERSION).tar.gz
+PKG_HASH:=0165867d25db434474001d26c122702a44f60e7f2ac017bdbd5a3cfa4a5ae6f3
+
+# Call original “make install” with prefix set to PKG_INSTALL_DIR
+PKG_INSTALL:=1
+
+# This is build/devel dependencies
+PKG_BUILD_DEPENDS:=cligen nghttp2 openssl
+
+include $(INCLUDE_DIR)/package.mk
+
+# Package definition; instructs on how and where our package will appear in the overall configuration menu ('make menuconfig')
+define Package/clixon
+	SECTION:=utils
+	CATEGORY:=Utilities
+	URL:=http://www.clixon.org
+	TITLE:=YANG-based toolchain
+	DEPENDS:=+cligen +libopenssl +libnghttp2 +libcurl \
+		 @(PACKAGE_openssh-server||PACKAGE_openssh-server-pam)
+	USERID:=clicon=61:clicon=61
+endef
+
+# Package description; a more verbose description on what our package does
+define Package/clixon/description
+	YANG-based toolchain including NETCONF and RESTCONF interfaces and an interactive CLI
+endef
+
+CONFIGURE_ARGS += \
+	--with-restconf=native \
+	--with-configfile=/etc/clixon.xml
+
+CONFIGURE_ARGS:=$(filter-out --disable-dependency-tracking,$(CONFIGURE_ARGS))
+
+define Build/Configure
+	$(call Build/Configure/Default, INSTALLFLAGS="")
+endef
+
+define Build/Compile
+	$(call Build/Compile/Default, DESTDIR="$(PKG_INSTALL_DIR)")
+endef
+
+# Source packages defining binary packages that ship shared libraries should declare a Build/InstallDev
+# recipe that copies all resources required to discover and link the shared libraries into the staging
+# directory.
+define Build/InstallDev
+	$(INSTALL_DIR) $(1)/usr/include
+	$(CP) $(PKG_INSTALL_DIR)/usr/include/clixon $(1)/usr/include/
+	$(INSTALL_DIR) $(1)/usr/lib/
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libclixon.so* \
+	      $(PKG_INSTALL_DIR)/usr/lib/libclixon_backend.so* \
+	      $(PKG_INSTALL_DIR)/usr/lib/libclixon_restconf.so* \
+	      $(PKG_INSTALL_DIR)/usr/lib/libclixon_cli.so* \
+	      $(1)/usr/lib/
+endef
+
+define Package/clixon/install
+	$(INSTALL_DIR) $(1)/etc/clixon
+	$(INSTALL_DIR) $(1)/usr/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/clixon_* $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/clixon_* $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/usr/share/clixon
+	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/usr/share/clixon/*.yang $(1)/usr/share/clixon/
+	$(INSTALL_DIR) $(1)/usr/lib
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libclixon.so* \
+	      $(PKG_INSTALL_DIR)/usr/lib/libclixon_backend.so* \
+	      $(PKG_INSTALL_DIR)/usr/lib/libclixon_restconf.so* \
+	      $(PKG_INSTALL_DIR)/usr/lib/libclixon_cli.so* \
+	      $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/etc/init.d
+	$(INSTALL_BIN) ./files/clixon.init $(1)/etc/init.d/clixon
+	$(INSTALL_DIR) $(1)/etc/ssh/sshd_config.d
+	$(INSTALL_DATA) -m 0600 ./files/netconf-subsystem.conf $(1)/etc/ssh/sshd_config.d/90-netconf-subsystem.conf
```

Comment:
INSTALL_DATA or INSTALL_CONF? 🤔 
See it here: https://openwrt.org/docs/guide-developer/packages#file_installation_macros